speed_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+NS=(11 23 27 29 35 37 41 43 47 51 57 65 87 103)
+MAIN_DIR="speed_test_results"
```

Comment:
Directory with results. Each one will have one directory with a log from the profiler and a file with an output of the program.